### PR TITLE
Gh actions run py3.10-3.12

### PR DESCRIPTION
**Pull request by patroqueeet**

### CODE REVIEW

**Comment by alexandernst** [comment]
might be time to drop 3.9 for the entire package 🤔

**Comment by patroqueeet** [comment]
@alexandernst any idea about the failing test?

File: README.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ This application implements the creation of **signed .pkpass** files and
 
 ## Requirements
 
-- Django 2.*, 3.*, 4.*
-- Python >= 3.6
+- Django 2.*, 3.*, 4.*, 5
```
Comment (alexandernst) [inline README.md]: I'm not sure django 2 and 3 make sense anymore since we require py3.10

**Comment by alexandernst** [comment]
looks like the zip file isn't being created correctly (¿?), but without further debugging I can't really asure it

**Comment by patroqueeet** [comment]
build fail: tries to extract empty file
```
['./', 'pass.json', 'manifest.json', 'signature']
  ^^^
```

where does it come from?

**Comment by patroqueeet** [comment]
@alexandernst ready to merge?

**Comment by alexandernst** [comment]
oh! didn't even manage to look properly at the zip issue! 😂

sure, let's merge!

**Pull request merged**